test/teiserver/protocols/spring/spring_auth_test.exs

```diff
@@ -591,6 +591,19 @@ CLIENTS test_room #{user.name}\n"
     reply = _recv_raw(socket)
     assert reply == "NO cmd=c.moderation.report_user\tdatabase error\n"
     assert Enum.count(Account.list_reports(search: [filter: {"target", target_user.id}])) == 2
+
+    # Reporting a friend
+    user = new_user()
+    %{socket: socket} = auth_setup(user)
+    %{user: friend} = auth_setup()
```

Comment:
You don't need to log new users in, we have an existing user. You can create a new user, add them to the existing user's friends and then do it. The login process is quite slow and one of the big reasons the tests can take a long time to run.